nvdaHelper/remote/ime.cpp

```diff
@@ -108,12 +108,12 @@ DWORD getIMEVersion(HKL kbd_layout, wchar_t* filename) {
 			// Do not know how to extract version number
 			return 0;
 	}
-	DWORD ver_handle;
+	DWORD ver_handle = 0;
 	DWORD buf_size = GetFileVersionInfoSizeW(filename, &ver_handle);
 	if (!buf_size)  return 0;
 	void* buf = malloc(buf_size);
     if (!buf)  return 0;
-	if (GetFileVersionInfoW(filename, ver_handle, buf_size, buf)) {
+	if (GetFileVersionInfoW(filename, /*ver_handle*/0, buf_size, buf)) {

```

Comment:
I find the inline comment confusing. MS docs state that the second parameter is not used. So we should just write 0, no matter what it may have been before this pr.

Reply:
The inline docs are for the sake of developers, to know which argument this is. The value passed in here is 0. I could use a `constexpr` instead, to give it a name?